Yoast/javascript
# Add Italian irregular diminutives

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a an exception check for irregular Italian diminutives to the Italian stemmer and removes _scrivere_, _scrive_, and _scritto_ from function words.

* [Yoast SEO Premium] Adds recognition of common irregular diminutives in Italian, such as nouns (e.g., ovetto - uovo), adjectives (e.g., dolciastro - dolce), and verbs (e.g., scribacchiare - scrivere). It also stops filtering out _scrivere_, _scrive_, and _scritto_ as function words during keyphrase analysis. 

## Technical choices

We added forms of the verb _scrivere_ (to write) to the exception list in this PR and noticed that some of the forms are not being recognised because they are in the function words list. After consulting with @iolandasequino they were removed from the function words as forms of _scrivere_ could plausibly be chosen as a keyword.

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all tests still pass.

* Check in the Italian Premium plugin. Try entering the following words as keyphrases: _scrivere_ , _uovo_, _dolce_. For each word, enter diminutive and non-diminutive forms of it in the text/title/meta description (scrivere -> scritto, scribacchiare; uovo -> uova, ovetto; dolce -> dolci, dolciastro). See whether they are recognised.

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-436
